runtime-sdk-typescript/index.ts

```diff
@@ -145,13 +145,25 @@ const main = async () => {
     account,
     { type: "local", input: { headless: false } },
   );
-  executeRuntimeScript({
+  await executeRuntimeScript({
     client,
     account,
     target: { browserContext: browserContext },
     initialUrl: appUrls[account.app],
     run: actions[account.app],
   });
+  const accountInfo = { ownerId: APP_USER_ID, domain: account.app };
+
+  // DEMONSTRATION of `getSessionStatus`, `deleteSession`
```

Comment:
DEMONSTRATION feels a bit strong
```suggestion
Demonstration
```

Reply:
🔊 🎺 all stand for DEMONSTRATION of our royal magistrate and his DECREE of CAMPAIGNING